src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -457,6 +457,10 @@ private function createTypeNotFoundMessage(TypedReference $reference, $label)
         } else {
             $message = $this->container->has($type) ? 'this service is abstract' : 'no such service exists';
             $message = sprintf('references %s "%s" but %s.%s', $r->isInterface() ? 'interface' : 'class', $type, $message, $this->createTypeAlternatives($reference));
+
+            if ($r->isInterface()) {
+                $message .= ' Did you write an implementation of this interface?';
```

Comment:
What about:

> Did you create a class that implements this interface?

Or:

> You may need to create a class that implements this interface.

Reply:
I like "Did you create a class that implements this interface?"